src/backend/InvenTree/InvenTree/settings.py

```diff
@@ -1336,11 +1331,12 @@
     'signup': 'InvenTree.forms.CustomSignupForm',
     'add_email': 'allauth.account.forms.AddEmailForm',
     'change_password': 'allauth.account.forms.ChangePasswordForm',
-    'set_password': 'allauth.account.forms.SetPasswordForm',
+    # 'set_password': 'allauth.account.forms.SetPasswordForm',
```

Comment:
Q (blocking): why is this changed?

Reply:
Right, I commented this out to get the code to launch - can we remove the custom 'set_password' form in `settings.py`?

Reply:
I do not think so but I would have to try to be certain